pkg/terminal/command.go

```diff
@@ -152,12 +152,17 @@ A list of file redirections can be specified after the new argument list to over
 	2>error.txt	redirects the standard error of the target process to error.txt
 `},
 		{aliases: []string{"rebuild"}, group: runCmds, cmdFn: c.rebuild, allowedPrefixes: revPrefix, helpMsg: "Rebuild the target executable and restarts it. It does not work if the executable was not built by delve."},
-		{aliases: []string{"continue", "c"}, group: runCmds, cmdFn: c.cont, allowedPrefixes: revPrefix, helpMsg: "Run until breakpoint or program termination."},
+		{aliases: []string{"continue", "c"}, group: runCmds, cmdFn: c.cont, allowedPrefixes: revPrefix, helpMsg: `Run until breakpoint or program termination.
+
+	continue [<linespec>]
+
+Optional linespec argument allows you to set a temporary breakpoint.
```

Comment:
We need to expand on this documentation for sure. I would give at least one example of usage and also explain why you would want to use it (e.g. essentially a "continue until" command). I'd also explain that any breakpoint hit between current location and the one specified will still cause the program to halt, so the user might not end up where they want to be.

Once #2208 lands maybe we can use it to toggle other breakpoints if that's desirable.

Reply:
@derekparker is this enough?

```
Run until breakpoint or program termination.

	continue [<linespec>]

Optional linespec argument allows you to continue until a specific location is reached. The program will halt if a breakpoint is hit before reaching the specified location.

For example:

	continue main.main
	continue encoding/json.Marshal
```

Reply:
Yes, that works!